engine-vuex/src/store.ts

```diff
@@ -62,10 +63,28 @@ export class WWTGlobalState {
   }
 }
 
+export class ImagesetInfo {
```

Comment:
The rest of the file doesn't do a great job of this, but we should try to include docstring comments in new definitions such as this structure and its fields.